lib/template.html

```diff
@@ -85,11 +85,20 @@
 {% endif %}
 
 {% if site.logo %}
-    {% assign seo_site_logo = site.logo | prepend: seo_url | escape %}
+  {% assign seo_site_logo = site.logo %}
+  {% unless seo_site_logo contains "://" %}
+    {% assign seo_site_logo = seo_site_logo | prepend: seo_url %}
+  {% endunless %}
+  {% assign seo_site_logo = seo_site_logo | escape %}
 {% endif %}
 
 {% if page.image %}
-    {% assign seo_page_image = page.image.path | default: page.image.facebook | default: page.image | prepend: seo_url | escape %}
+  {% assign seo_page_image = page.image.path | default: page.image.facebook |
+default: page.image %}
```

Comment:
In the process of changing text editors; didn't notice it was hard wrapping this :rage2: